pkg/pattern/drain/drain_test.go

```diff
@@ -91,7 +91,7 @@ func TestDrain_TrainExtractsPatterns(t *testing.T) {
 			},
 		},
 		{
-			drain:     New(DefaultConfig(), "", nil),
+			drain:     New("", DefaultConfig(), &fakeLimits{}, "", nil),
```

Comment:
nit: define a tenantID constant for this, so its obvious what is being passed here

Reply:
Done